legacy/setup-debian.sh

```diff
@@ -4,11 +4,12 @@ export DEBIAN_FRONTEND=noninteractive
 apt-get update
 apt-get -y upgrade
 apt-get -y install autoconf automake binutils \
-bison build-essential bzip2 ca-certificates cmake cmake-data coreutils \
+bison build-essential bzip2 ca-certificates cmake-data coreutils \
```

Comment:
```suggestion
bison build-essential bzip2 ca-certificates coreutils \
```
Just remove cmake completely. cmake-data is pulled in by cmake anyway.